File: app/src/main/java/ir/arapp/arappmain/Util/Services/NavigationManager.java
package ir.arapp.arappmain.Util.Services;

public interface NavigationManager
{
//    Lock drawer in some fragments
    void setDrawerLocked(boolean shouldLock);
//    Open Drawer when Nav Icon clicked
    void openDrawer();
//    Bottom Navigation manager
    void bottomNavigationVisibility(boolean setVisible);
}

File: app/src/main/java/ir/arapp/arappmain/util/services/NavigationManager.kt
package ir.arapp.arappmain.util.services

interface NavigationManager {
    //    Lock drawer in some fragments
    fun setDrawerLocked(shouldLock: Boolean)

    //    Open Drawer when Nav Icon clicked
    fun openDrawer()

    //    Bottom Navigation manager
    fun bottomNavigationVisibility(setVisible: Boolean)
}